docs/UserGuide.md

```diff
@@ -102,87 +92,147 @@ There are three main areas in TutorBuddy:
+  * **`add_student`**`n/John Doe p/98765432 e/johnd@example.com a/John street, Block 123, #01-01 l/Sec2 g/95421323 r/Mother` : Adds a student named `John Doe`
     to TutorBuddy.
-
   * **`delete_student`**`3` : Deletes the 3rd student shown in the Student section.
-
   * **`add_session`**`n/John Doe d/2021-01-01 t/13:00 k/120 s/Biology f/80`: Adds a single tuition session for John Doe happening on 2021-01-01
-
   * **`add_recurring_session`**`n/John Doe d/2021-01-01 t/18:00 k/120 s/Biology f/80 b/7 e/2021-01-15`: Adds a 7-day recurring session for John Doe happening from 2021-01-01 to 2021-01-15
+  * **`exit`** : Exits the application.<br>
+    Refer to the [Commands](#commands) below for details of each command.
 
-  * **`exit`** : Exits the application.
+--------------------------------------------------------------------------------------------------------------------
 
-  * Refer to the [Features](#features) below for details of each command.
+## Commands
 
---------------------------------------------------------------------------------------------------------------------
+Commands are the main way you interact with TutorBuddy.
+
+This section provides information about all commands that is available in TutorBuddy.
 
-## Features
+### Command Format
 
 <div markdown="block" class="alert alert-info">
 
 **:information_source: Notes about the command format:**<br>
 
+This section details the format of the commands available in TutorBuddy. We will adhere to the following:
+
+
 * Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
   e.g. in `add_student n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
 
 * Items in square brackets are optional.<br>
-  e.g `find_student KEYWORD [MORE_KEYWORDS]` can be used as `find_student John Samuel` or as `find_student John`.
+  e.g. `find_student KEYWORD [MORE_KEYWORDS]` can be used as `find_student John Samuel` or as `find_student John`.
 
 * Parameters can be in any order.<br>
   e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
 
 * If a parameter is expected only once in the command but you specified it multiple times, only the last occurrence of the parameter will be taken.<br>
-  e.g. if you specify `p/12341234 p/56785678`, only `p/56785678` will be taken.
+  e.g. if you specify `p/98765432 p/99999999`, only `p/98765432` will be taken.
 
-* Extraneous parameters for commands that do not take in parameters (such as `emails`, `help`, `exit` and `clear`) will be ignored.<br>
+* Extraneous parameters for commands that do not take in parameters (such as `emails`, `help`, `exit`, `clear` and `list_student`) will be ignored.<br>
   e.g. if the command specifies `help 123`, it will be interpreted as `help`.
 
+* Indexes **must be positive whole numbers** 1, 2, 3, …
+
+* The `STUDENT_INDEX` refers to the index number shown in the [displayed student list](#application-overview).
```

Comment:
The link here not working on mine

Reply:
As with the following displayed session list

Reply:
Works fine for me, probably some browser issue?

Reply:
Riding onto this thread for another reason: should it be indices instead of indexes? HAHA

Reply:
i googled and it says both also can but "indexes" is a more English way while "indices" is a more latin way haha